audio/FAudio/Portfile

```diff
@@ -30,6 +29,19 @@ configure.args          -DBUILD_UTILS=OFF \
                         -DXNASONG=ON \
                         -DCMAKE_INSTALL_INCLUDEDIR=include/FAudio
 
+# FAudio has a hard requirment of SDL 2.24, since 22.09
+if {${os.major} <= 10} {
+    known_fail yes
+    archive_sites
+    distfiles
+    depends_build
+    depends_lib
+    pre-fetch {
+        ui_error "${name} @${version} requires Mac OS X 10.7 or later."
+        return -code error "incompatible Mac OS X version"
+    }
+}
```

Comment:
instead of doing this please just restrict it the to latest version that still works